tests_e2e/orchestrator/scripts/install-agent

```diff
@@ -167,8 +167,21 @@ fi
 echo "Restarting service..."
 agent-service stop
 
+# Agent log may not be present if install-agent setup run before pre-install agent start; So giving it a few minutes to show up
```

Comment:
The stop will not throw error because the service file exist that comes part of image. So that will work. The case I have seen, start and stop happened around same time. So that lead to log file not created. 

2024-03-08T09:24:04.273995+00:00 localhost sudo:  waagent : TTY=unknown ; PWD=/home/waagent ; USER=root ; COMMAND=/usr/bin/env PATH=/home/waagent/bin:/usr/local/bin:/usr/bin:/bin:/usr/lib/mit/bin PYTHONPATH=/home/waagent/lib install-agent --package /home/waagent/tmp/WALinuxAgent-9.9.9.9.zip --version 9.9.9.9
2024-03-08T09:24:04.275404+00:00 localhost sudo: pam_unix(sudo:session): session opened for user root by (uid=1000)
2024-03-08T09:24:04.402819+00:00 localhost systemd[1]: Started Obtain Cloud SMT info and register with the SMT server.
**2024-03-08T09:24:04.403041+00:00 localhost systemd[1]: Started Azure Linux Agent.**
2024-03-08T09:24:04.403171+00:00 localhost systemd[1]: Reached target Multi-User System.
2024-03-08T09:24:04.403284+00:00 localhost systemd[1]: Reached target Graphical Interface.
2024-03-08T09:24:04.403404+00:00 localhost systemd[1]: Starting Update UTMP about System Runlevel Changes...
2024-03-08T09:24:04.403517+00:00 localhost systemd[1]: Starting Execute cloud user/final scripts...
2024-03-08T09:24:04.417516+00:00 localhost systemd[1]: Started Update UTMP about System Runlevel Changes.
2024-03-08T09:24:05.187337+00:00 localhost ec2: 
2024-03-08T09:24:05.187575+00:00 localhost ec2: #############################################################
2024-03-08T09:24:05.187702+00:00 localhost ec2: -----BEGIN SSH HOST KEY FINGERPRINTS-----
2024-03-08T09:24:05.188070+00:00 localhost ec2: #############################################################
**2024-03-08T09:24:05.246741+00:00 localhost systemd[1]: Stopping Azure Linux Agent...
2024-03-08T09:24:05.247947+00:00 localhost systemd[1]: Stopped Azure Linux Agent.**

Reply:
2024-03-08T09:24:04.403041+00:00 localhost systemd[1]: Started Azure Linux Agent.
This line indicates when service started, before deamon creates file, service got stopped. Anyway, I'll send you the log offline.